lxd/container_lxc.go

```diff
@@ -5642,27 +5639,16 @@ func (c *containerLXC) Update(args db.ContainerArgs, userRequested bool) error {
 		for k, m := range updateDevices {
 			if m["type"] == "disk" {
 				updateDiskLimit = true
-			} else if m["type"] == "nic" || m["type"] == "infiniband" {
```

Comment:
@stgraber this is the part I mentioned about the infiniband reference that doesn't appear to do anything to do with infiniband anymore.